sklearn/tests/test_naive_bayes.py

```diff
@@ -339,9 +345,13 @@ def test_discretenb_provide_prior(DiscreteNaiveBayes):
     assert_array_almost_equal(prior, np.array([.5, .5]))
 
     # Inconsistent number of classes with prior
-    assert_raises(ValueError, clf.fit, [[0], [1], [2]], [0, 1, 2])
-    assert_raises(ValueError, clf.partial_fit, [[0], [1]], [0, 1],
-                  classes=[0, 1, 1])
+    msg_1 = re.escape('Number of priors must match number of classes')
+    with pytest.raises(ValueError, match=msg_1):
+        clf.fit([[0], [1], [2]], [0, 1, 2])
+
+    msg_2 = re.escape('is not the same as on last call to partial_fit')
+    with pytest.raises(ValueError, match=msg_2):
+        clf.partial_fit([[0], [1]], [0, 1], classes=[0, 1, 1])
```

Comment:
There is no need to use two different variables `msg_1` and `msg_2` here, I would be slightly in favor of using `msg` as everywhere else...

Reply:
Thanks for the review @cmarmo. Implemented the suggested changes in the last commit!